Running main() from gtest_main.cc
Note: Randomizing tests' orders with a seed of 53637 .
[==========] Running 6 tests from 1 test case.
[----------] Global test environment set-up.
[----------] 6 tests from StockProfit
[ RUN      ] StockProfit.Get_max_profit_after_buy_for_two_day_prices
StockProfit.tests.cpp:70: Failure
Value of: stockProfit.getMaxProfitsAfterBuy()
  Actual: {}
Expected: expectMaxProfitsAfterBuy
Which is: { -1, -2 }
[  FAILED  ] StockProfit.Get_max_profit_after_buy_for_two_day_prices (9 ms)
[ RUN      ] StockProfit.Get_max_profit_after_buy_for_one_day_prices
[       OK ] StockProfit.Get_max_profit_after_buy_for_one_day_prices (0 ms)
[ RUN      ] StockProfit.One_day_prices
[       OK ] StockProfit.One_day_prices (0 ms)
[ RUN      ] StockProfit.Empty_prices
[       OK ] StockProfit.Empty_prices (0 ms)
[ RUN      ] StockProfit.Get_max_profit_after_buy_for_empty_prices
[       OK ] StockProfit.Get_max_profit_after_buy_for_empty_prices (0 ms)
[ RUN      ] StockProfit.Two_day_prices
[       OK ] StockProfit.Two_day_prices (0 ms)
[----------] 6 tests from StockProfit (11 ms total)

[----------] Global test environment tear-down
[==========] 6 tests from 1 test case ran. (13 ms total)
[  PASSED  ] 5 tests.
[  FAILED  ] 1 test, listed below:
[  FAILED  ] StockProfit.Get_max_profit_after_buy_for_two_day_prices

 1 FAILED TEST
make: *** [test.output] Error 1
